cluster/gce/util.sh

```diff
@@ -1728,6 +1737,8 @@ function kube-down() {
       "${NETWORK}-default-internal"  # Pre-1.5 clusters
 
     if [[ "${KUBE_DELETE_NETWORK}" == "true" ]]; then
+      # Delete all remaining firewall rules in the network.
+      delete-all-firewall-rules || true
```

Comment:
Maybe call this inside `delete-network` and get rid of the `# might fail if there are leaked firewall rules` comment below?

Reply:
Ref https://github.com/kubernetes/kubernetes/pull/53626#discussion_r143827108, WDYT about keeping resource cleanup detached?

I revised that comment a bit as it isn't completely right. Network deletion might fail no only due to firewalls, but also routes, VMs, or any others that reference this network.

Reply:
Sounds good.